pr_check.sh

```diff
@@ -3,24 +3,26 @@
 # --------------------------------------------
 # Options that must be configured by app owner
 # --------------------------------------------
-APP_NAME="patchman"  # name of app-sre "application" folder this component lives in
-COMPONENT_NAME="patchman"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
-IMAGE="quay.io/cloudservices/patchman-engine-app"
-DOCKERFILE="Dockerfile.rhel8"
-COMPONENTS_W_RESOURCES="vmaas"
+export APP_NAME="patchman"  # name of app-sre "application" folder this component lives in
+export COMPONENT_NAME="patchman"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
+export IMAGE="quay.io/cloudservices/patchman-engine-app"
+export DOCKERFILE="Dockerfile.rhel8"
+export COMPONENTS_W_RESOURCES="patchman"
```

Comment:
this is not correct
`bonfire` by default removes resource requests/limits from clowdapp.yaml to avoid problems with resource over-commitment because a lot of teams have their resources set way too high (resource limits/requests defined by ClowdEnvironment are used instead). `COMPONENTS_W_RESOURCES` is meant to deploy required app to ephemeral env respecting the requests/limits set in app's clowdapp.yaml. It basically does `bonfire deploy ... --no-remove-resources $COMPONENTS_W_RESOURCES`

Default resource/limits in ClowdEnvironment aren't high enough for `vmaas` to sync and unpack repositories, therefore we must respect resources set in vmaas clowdapp.yaml and use `COMPONENTS_W_RESOURCES="vmaas"` (see https://github.com/RedHatInsights/patchman-engine/pull/727)

Don't set `COMPONENTS_W_RESOURCES="patchman"` until you are 100% sure that default resources in ephemeral ClowdEnvironment are not high enough for patchman and that resource limits/requests are set to really minimal values required by the patchman in clowdapp.yaml

Reply:
it makes sense. Thanks!